cosmoz-image-viewer.js

```diff
@@ -5,199 +5,67 @@
 
 	Polymer({
 		behaviors: [
-			Polymer.IronResizableBehavior,
 			Cosmoz.TemplateHelperBehavior,
 			Cosmoz.TranslatableBehavior
 		],
 		is: 'cosmoz-image-viewer',
 		properties: {
-			currentImage: {
-				type: Object,
-				observer: '_currentImageChanged'
-			},
-			currentImageIndex: {
-				type: Number,
-				computed: '_computeCurrentImageIndex(currentImage, images)'
-			},
-			currentPage: {
-				type: Number,
-				computed: '_computePage(currentImageIndex)'
-			},
-			detached: {
-				type: Boolean,
-				value: false,
-				readOnly: true,
-				notify: true,
-				observer: '_detachedChanged'
-			},
-			imageLoaded: {
-				type: Boolean,
-				value: false
-			},
 			images: {
 				type: Array,
-				observer: '_imageListChanged'
-			},
-			/**
-			 * Disable button to detach image
-			 */
-			noDetach: {
-				type: Boolean,
-				value: false
-			},
-			scrollPercent: {
```

Comment:
Not part of PhotoSwipe, it's a setting to make the image scroll on load (for certain documents we try to scroll down past the header).